docroot/themes/humsci/humsci_basic/patterns/vertical-card/vertical-card.ui_patterns.yml

```diff
@@ -23,17 +23,6 @@ vertical_card:
       label: "Category"
       preview: "Category"
       type: text
-    url:
-      label: "URL"
-      preview: "URL to the item"
-      type: text
-    # The following are used in the original theme but not here
-    author:
```

Comment:
we can remove author and year?

Reply:
(like is H&S ok with that)?

Reply:
I added them back in. It doesn't hurt anything and it may be nice to have for future reference.